src/resources/markdown_worker.ts

```diff
@@ -22,8 +22,10 @@ const renderMarkdown = (
   if (!whiteListNormal) {
     whiteListNormal = {
       ...(getDefaultWhiteList() as WhiteList),
+      style: [],
```

Comment:
We can't allow every style, it allows for xss

Reply:
Yeah, we might need something like this: https://github.com/leizongmin/js-css-filter

Same author as the `xss` module we already use, but quite old.

Reply:
> 
> 
> Yeah, we might need something like this: https://github.com/leizongmin/js-css-filter
> 
> Same author as the `xss` module we already use, but quite old.

I'll separate the <style> part into a separate future PR.